atlasdb-ete-test-utils/src/main/java/com/palantir/atlasdb/ete/Gradle.java

```diff
@@ -42,10 +42,10 @@ public static Gradle ensureTaskHasRun(String command) {
     @Override
     protected void before() throws Throwable {
         if (isRunningOutsideOfGradle()) {
-            System.out.println("It looks like you are not running in gradle, performing the required gradle command: " + command);
+            System.out.println("It looks like you are not running in gradle, performing the required gradle command: " + command);  // (authorized)
```

Comment:
This is a utility class that helps determine whether or not a necessary gradle task has been run to kick off testing.  It allows you to run the test in intelij but use the same dependent gradle tasks that can't be easily replicated in intelij @joelea who set this up